rails/empty-arrays-in-json-request.md

```diff
@@ -0,0 +1,66 @@
+Deep Munging
```

Comment:
Right, I think a short sentence and a link will do.

It might even make sense to not mention it at all, since we found that these hoops exist even when its disabled.